### PR TITLE
feat: add pre-commit check check-chinese and change rev

### PR DESCRIPTION
**Pull request by CodeChenL**
###### Description of changes
* close https://github.com/radxa-docs/pre-commit-hooks/issues/6

###### Review guide

To review this PR locally, please run the following commands:

```bash
PR= # the current PR number
git fetch origin pull/${PR}/head
git switch --detach FETCH_HEAD
yarn start
yarn start --locale en
# If you want to start 2 locales side-by-side, use the following command on Linux:
# https://github.com/facebook/docusaurus/issues/7377#issuecomment-1167192715
yarn start & DOCUSAURUS_GENERATED_FILES_DIR_NAME=.docusaurus/en yarn start --locale en --port 3001
kill %1
```

### CODE REVIEW

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -17,13 +17,15 @@ repos:
       - id: index-to-README
       - id: globally-available-components
       - id: remove-category-json
+      - id: check-chinese
+        files: "i18n/en/docusaurus-plugin-content-docs/current"
   - repo: https://github.com/detailyang/pre-commit-shell
-    rev: v1.0.6
+    rev: 1.0.5
```
Comment (RadxaYuntian) [inline .pre-commit-config.yaml:23]:
```suggestion
    rev: v1.0.6
```

File: .pre-commit-config.yaml [hunk 2]
```diff
@@ -45,13 +47,13 @@ repos:
...
   - repo: https://github.com/pre-commit/mirrors-prettier
-    rev: v3.1.0
+    rev: v4.0.0-alpha.8
     hooks:
       - id: prettier
         stages:
           - pre-commit
   - repo: https://github.com/crate-ci/typos
-    rev: v1.16.23
+    rev: typos-v0.10.21
```
Comment (RadxaYuntian) [inline .pre-commit-config.yaml:56]:
```suggestion
    rev: v1.18.2
```

File: .pre-commit-config.yaml [hunk 3]
```diff
@@ -45,13 +47,13 @@ repos:
       - id: trailing-whitespace
         args: [--markdown-linebreak-ext=md]
   - repo: https://github.com/pre-commit/mirrors-prettier
-    rev: v3.1.0
+    rev: v4.0.0-alpha.8
```
Comment (RadxaYuntian) [inline .pre-commit-config.yaml:50]:
```suggestion
    rev: v3.1.0
```

File: .pre-commit-config.yaml [hunk 4]
```diff
@@ -17,13 +17,15 @@ repos:
...
+        files: "i18n/en/docusaurus-plugin-content-docs/current"
   - repo: https://github.com/detailyang/pre-commit-shell
-    rev: v1.0.6
+    rev: 1.0.5
     hooks:
       - id: shell-lint
         args: [-x]
   - repo: https://github.com/pre-commit/mirrors-prettier
-    rev: v3.1.0
+    rev: v4.0.0-alpha.8
```
Comment (RadxaYuntian) [inline .pre-commit-config.yaml:28]:
```suggestion
    rev: v3.1.0
```

**Review by RadxaYuntian — CHANGES_REQUESTED** [review]
注意点不要逆向升级

File: .github/CODEOWNERS [hunk 5]
```diff
@@ -58,3 +58,4 @@ i18n/en/docusaurus-plugin-content-docs/current/rock3/rock3a/os-config/rsetup.md
 static/img/common/etcher/rock5a-step3.webp @u7985
 docs/compute-module/nx5/radxa-os/apps-deploy/README.md @peterwang2050
 i18n/en/docusaurus-plugin-content-docs/current/nio/nio12l/download/README.md @peterwang2050
+.pre-commit-config.yaml @CodeChenL
```
Comment (RadxaYuntian) [inline .github/CODEOWNERS:61]:
```suggestion
.pre-commit-config.yaml @CodeChenL @RadxaYuntian 
```
然后移动到上面CI/CD section里面

File: .pre-commit-config.yaml [hunk 6]
```diff
@@ -17,6 +17,8 @@ repos:
       - id: index-to-README
       - id: globally-available-components
       - id: remove-category-json
+      - id: check-chinese
+        files: "i18n/en/docusaurus-plugin-content-docs/current"
```
Comment (RadxaYuntian) [inline .pre-commit-config.yaml:21]: 我们应该有现有文件无法通过这个测试。在你本地github仓库确认一下pr时只会检查pr里面涉及的commit

**Review by RadxaYuntian — CHANGES_REQUESTED** [review]

File: .pre-commit-config.yaml (hunk 6 above)
Comment (CodeChenL) [reply .pre-commit-config.yaml:21]: ci只会检查pr里面涉及的commit

**Review by RadxaYuntian — COMMENTED** [review]
squash

**Review by RadxaYuntian — APPROVED** [review]

**Pull request merged**